extractor/filesystem/osv/metadata.go

```diff
@@ -21,3 +21,27 @@ type Metadata struct {
 	Ecosystem string
 	CompareAs string
 }
+
+// DepGroups provides access to the dependency groups property in some metadata objects
```

Comment:
I don't really understand what this is used for based on this comment, can you expand on what a "dependency group" is?

Reply:
Updated this comment to be clearer.